continuous-integration.md

```diff
@@ -0,0 +1,31 @@
+Continuous Integration
+====
+
+Continuous Integration (CI) is a practice where developers frequently push their code into a shared repository. 
+Automated builds and checks run on each check-in, allowing teams to detect problems early.
+
+## Tasks
+
+You should have continuous integration set up in your repository running the following tasks where relevant:
+* Running unit and integration tests
```

Comment:
Should we make a note of UI / visual regression testing here? The interface of our applications is often an important part of the integration process.